tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/merge.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import PageRegion
+
+
+class Merge(PageRegion):
+
+    _iframe_locator = (By.CSS_SELECTOR, '.popupWindow.active iframe[data-url*="matching"]')
+    _merge_button = (By.ID, 'merge-action')
+
+    def __init__(self, marionette):
+        marionette.switch_to_frame()
+        
+        root = Wait(marionette).until(
+            expected.element_present(*self._iframe_locator))        
+        Wait(marionette).until(expected.element_displayed(root))
+        marionette.switch_to_frame(root)
+
```

Comment:
Coordinates in a rectangle are defined based on the top-left corner. As a consequence, y=0 represent the top of the screen.